### PR TITLE
Change blanknodes to namednodes

### PR DESCRIPTION
**Pull request by nlisgo**
The aim of this PR is to introduce the smallest amount of changes to swap out `BlankNodes` for `NamedNodes`.

### CODE REVIEW

**Comment by nlisgo** [comment]
@giorgiosironi I would like this to be considered for release. I know it doesn't address all of the todo items but it adds enough value in my opinion. The next piece of work will be to improve the identifier to include the `/articles/` prefix and then we will want to allow the retrieval of a single article in the store. 

I believe this will be enough to unblock the database persistence work.

File: src/routes/add-article.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default (): AppMiddleware => (
       throw new createHttpError.BadRequest(`Article must have at least one ${termToString(schema('name'))}`);
     }
 
-    const newId = blankNode(uniqueString());
+    const newId = namedNode(uniqueString());
```
Comment (giorgiosironi) [inline src/routes/add-article.ts:31]: The documentation says [this should be an IRI](http://rdf.js.org/data-model-spec/#namednode-interface), but we have [good reasons to use host-free values](
https://github.com/libero/publisher/issues/367#issuecomment-574638754).

**Review by giorgiosironi — APPROVED** [review]
Checking NamedNode basic requirements I think this should go ahead.
The other tasks in https://github.com/libero/publisher/issues/367 however still need to be followed up before persistence can be introduced as the whole ticket has been created as a blocker (also I see that at the bare minimum identifiers like `/articles/...` should be stable before they start to get persisted).

**Comment by giorgiosironi** [comment]
Found out this couldn't be merged by @nlisgo because the [branch protection on `master`](https://github.com/libero/article-store/settings/branch_protection_rules/11906254) limits pushing to `Organization administrators, repository administrators, and users with the Maintain role` plus Dependabot. https://github.com/orgs/libero/teams/node-js-reviewers only has `Write`, not `Maintain` permission.

**Pull request merged**